Source/HelixToolkit.Wpf/Importers/StudioReader.cs

```diff
@@ -547,7 +547,7 @@ private void ReadMaterial(BinaryReader reader, int chunkSize)
                         }
 
                         mg.Children.Add(new SpecularMaterial(new SolidColorBrush(specular), specularPower));
-
+                        mg.Children.Add(new EmissiveMaterial((new SolidColorBrush( luminance))));

```

Comment:
Maybe I was to fast with the pull request, I'll add an if clause. Anyway this was just a quirky workaroun, as I use the HelixToolkit.WPF.StudioImport in the Sharp.DX extension. Basically just converting the resulting WPF MeshGeometry3D into the according SharpDX type. So I should add a proper handling for this either by
1. Adding a proper Converter from WPF MeshGeometry3d to SharpDX 
or
2. Port the StudioImporter to HelixToolkit.Wpf.SharpDX

Sorry for the inconvenience.